pyensembl/exon.py

```diff
@@ -35,12 +35,6 @@ def __str__(self):
         return "Exon(exon_id=%s, gene_name=%s, contig=%s, start=%d, end=%s)" % (
             self.id, self.gene_name, self.contig, self.start, self.end)
 
-    def __repr__(self):
```

Comment:
All of our implementations just call `str` so I made that the default in `Serializable`. Maybe that object needs to be renamed to something like `SaneDefaultsForClassesWhichImplementToDict`.